server/api/scraper.py

```diff
@@ -0,0 +1,49 @@
+import re
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from webdriver_manager.chrome import ChromeDriverManager
+from webdriver_manager.utils import ChromeType
```

Comment:
Make sure to do `pipenv install [LibraryName]` so that your pipfile is updated aswell (you might have only installed it the packages in your local environment)